Fixed error message prints function decorators when using assert in Python 3.8 and above.
